extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientSupport.java

```diff
@@ -31,6 +45,22 @@ public MongoClientSupport(List<String> codecProviders, List<String> bsonDiscrimi
         return connectionPoolListeners;
     }
 
+    public List<CommandListener> getCommandListeners() {
+        return commandListeners;
+    }
+
+    public void setCommandListenerList(List<CommandListener> commandListenersList) {
```

Comment:
I would drop this method, it's useless, right?

Reply:
It depends on the above comment (https://github.com/quarkusio/quarkus/pull/12082#discussion_r488011504). If we need to add a single CommandListener or a full list. But finally it should exist only the one that it's used.